content/blog/2022-02-22-profiling.md

```diff
@@ -0,0 +1,208 @@
++++
+title = "Efficient Path Profiling"
+[[extra.authors]]
+name = "Niansong Zhang"
+link = "https://www.zzzdavid.tech"
+[[extra.authors]]
+name = "Jiajie Li"
+link = "https://tonyjie.github.io/"
++++
+
+<p align="center">
+<img src="https://res.cloudinary.com/dxzx2bxch/image/upload/v1646243822/cs6120/2_ugrqoi.png" alt="alt_text" title="image_tooltip" style="zoom:25%;" />
```

Comment:
It's possible to include your image right here instead of using a third-party hosting service. Just make a directory instead of a single `.md` file, like this: https://github.com/sampsyo/cs6120/tree/2020fa/content/blog/2020-11-09-modern-gc